stl/inc/yvals_core.h

```diff
@@ -1296,6 +1292,10 @@
 #define __cpp_lib_shared_ptr_arrays 201611L // P0497R0 Fixing shared_ptr For Arrays
 #endif // _HAS_CXX20
 
+#if defined(__cpp_impl_coroutine) || defined(_DOWNLEVEL_COROUTINES_SUPPORTED) // TRANSITION, Clang coroutine support

```

Comment:
Late feedback: We should remove the `TRANSITION` comment. Now that this is outside of `_HAS_CXX20`, this condition is permanent. I hate to make a PR for a single comment cleanup. @STL, do you have a "miscellaneous cleanups" branch laying around you could add this change to, or should I start one of my own?

Reply:
I had recorded a todo about the matching comment being out of date (too small to reset testing), I’ll add this to that list, and fix it when we’re done with deadlines.